docs/api/rest_api_authentication.md

```diff
@@ -0,0 +1,74 @@
+# REST API reference authentication basics
+
+This page refers to [REST API reference](https://doc.ezplatform.com/rest-api-reference), where you can find detailed information about
+REST API resources and endpoints.
+
+Use of HTTPS for authenticated (REST) traffic is highly recommended.
+
+## Basic Authentication
+
+For more information, see [HTTP Authentication: Basic and Digest Access Authentication.](http://tools.ietf.org/html/rfc2617)
+
+## OAuth
+
+For more information, see [OAuth 2.0 protocol for authorization.](https://oauth.net/2/)
+
+## Session based Authentication
+
+The sessions are only created to re-authenticate the user (and perform authorization), not to hold session state in the service.
+Because of that, we consider this method supports AJAX based applications even if it violates the principles of RESTful services.
+
+For more information, see [REST API authentication](general_rest_usage.md#rest-api-authentication).
+
+### Session cookie
+
+If activated, the user must log in, and the client must send the session cookie in every request, using standard Cookie header.
+The name (`sessionName`) and value (`sessionID`) of the header are defined  in a `/user/sessions` POST response.
+
+Example request header: `Cookie: <SessionName> : <sessionID>`.
+
+### CSRF token
+
+A CSRF token needs to be sent in every request that uses "unsafe" methods (not GET or HEAD), when a session has been established.
+It should be sent with a `X-CSRF-Token` header.
+The token (`csrfToken`) is defined in a response during logging in through the POST `/user/sessions`.
+
+Example request headers:
+
+```
+DELETE /content/types/32 HTTP/1.1
+X-CSRF-Token: <csrfToken>
+```
+
+```
+DELETE /user/sessions/<sessionID>
+X-CSRF-Token: <csrfToken>
+```
+
+If an unsafe request is missing the CSRF token, or the token has incorrect value, an error is returned: `401 Unauthorized`.
+
+### Rich client application security concerns
+
+The purpose of CSRF protection is to prevent users accidentally running harmful operations by being tricked into executing an http(s) request against a web applications they are logged into.
```

Comment:
```suggestion
The purpose of CSRF protection is to prevent users from accidentally running harmful operations by being tricked into executing an http(s) request against a web applications they are logged into.
```